src/app/content/components/Page/contentDOMTransformations.ts

```diff
@@ -1,10 +1,11 @@
-import { HTMLButtonElement, HTMLElement } from '@openstax/types/lib.dom';
+import { Document, HTMLButtonElement, HTMLElement } from '@openstax/types/lib.dom';
 import { IntlShape } from 'react-intl';
 import { assertDefined, assertNotNull } from '../../../utils';
 
 // from https://github.com/openstax/webview/blob/f95b1d0696a70f0b61d83a85c173102e248354cd
 // .../src/scripts/modules/media/body/body.coffee#L123
-export const transformContent = (rootEl: HTMLElement, intl: IntlShape) => {
+// We are passing Document because it is required to prerender.
+export const transformContent = (_document: Document, rootEl: HTMLElement, intl: IntlShape) => {
```

Comment:
you're passing document but you're not using it for anything, you need to pass the reference down so it is used in places like this https://github.com/openstax/rex-web/pull/449/files#diff-3684e43af97114ebe9e858e13d39add5R56

Reply:
Omg, now this makes sense. I could sware that `document` was not used in this file (I had to mistype it when I was searching) and I thought that prerendering script is somehow getting access to this...